client/state/selectors/get-current-plan-term.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Internal dependencies
+ */
+
+import { getPlan } from 'lib/plans';
+import { getSitePlan } from 'state/sites/selectors';
+
+/**
+ * Returns term of the active plan for given siteId, e.g. TERM_MONTHLY
```

Comment:
This could potentially deceive someone to expect literally `TERM_MONTHLY`, while the actual value they should expect is `monthly`.